src/views/Pools/components/PoolsTable/Cells/EarningsCell.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Skeleton, Text, useTooltip, HelpIcon, Flex, Box, useModal } from '@pancakeswap/uikit'
+import { Pool } from 'state/types'
+import BigNumber from 'bignumber.js'
+import { PoolCategory } from 'config/constants/types'
+import { BIG_ZERO } from 'utils/bigNumber'
+import { formatNumber, getBalanceNumber, getFullDisplayBalance } from 'utils/formatBalance'
+import Balance from 'components/Balance'
+import { useCakeVault, useBusdPriceFromToken } from 'state/hooks'
+import { useTranslation } from 'contexts/Localization'
+import { convertSharesToCake } from 'views/Pools/helpers'
+import BaseCell, { CellContent } from './BaseCell'
+import CollectModal from '../../PoolCard/Modals/CollectModal'
+
+interface EarningsCellProps {
+  pool: Pool
+  account: string
+  userDataLoaded: boolean
+  isAutoVault?: boolean
+}
+
+const StyledCell = styled(BaseCell)`
+  flex: 4.5;
```

Comment:
Why 4.5?

Reply:
So there is bunch of weird flex ratios here and there, this is mostly fine-tuned in order to show table properly on all screen sizes. Whole numbers are nice but as you resize the window it looks awful at certain resolutions.
Even if there is solution for that that can use whole numbers it would probably require other complex stuff around it.
And non-integer flex values are supported in every browser as far as I know (maybe except IE11) so shouldn't cause any issues 🙂